### PR TITLE
Add arity to functions in docs

### PR DESCRIPTION
**Pull request by eksperimental**

### CODE REVIEW

File: lib/elixir/lib/macro.ex [hunk 1]
```diff
@@ -291,7 +291,7 @@ defmodule Macro do
   into a syntax tree.
 
   One may pass `unquote: true` to `escape/2`
-  which leaves `unquote` statements unescaped, effectively
+  which leaves `unquote/2` statements unescaped, effectively
```
Comment (lexmag) [inline lib/elixir/lib/macro.ex]: `unquote` has arity 1.

File: lib/ex_unit/lib/ex_unit/doc_test.ex [hunk 2]
```diff
@@ -175,7 +175,7 @@ defmodule ExUnit.DocTest do
 
     * `:import` - when `true`, one can test a function defined in the module
       without referring to the module name. However, this is not feasible when
-      there is a clash with a module like Kernel. In these cases, `import`
+      there is a clash with a module like Kernel. In these cases, `import/2`
```
Comment (lexmag) [inline lib/ex_unit/lib/ex_unit/doc_test.ex]: In this case we're talking about `:import` option.

File: lib/elixir/lib/kernel/special_forms.ex [hunk 3]
```diff
@@ -1550,7 +1550,7 @@ defmodule Kernel.SpecialForms do
   be used to control flow based on the result of the expression.
   Catch, rescue and else clauses work based on pattern matching.
 
-  Note that calls inside `try` are not tail recursive since the VM
+  Note that calls inside `try/2` are not tail recursive since the VM
```
Comment (lexmag) [inline lib/elixir/lib/kernel/special_forms.ex]: `try` has arity 1.

File: lib/elixir/lib/kernel/special_forms.ex [hunk 4]
```diff
@@ -768,7 +768,7 @@ defmodule Kernel.SpecialForms do
     * `:context` - sets the resolution context.
 
     * `:bind_quoted` - passes a binding to the macro. Whenever a binding is
-      given, `unquote` is automatically disabled.
+      given, `unquote/2` is automatically disabled.
```
Comment (lexmag) [inline lib/elixir/lib/kernel/special_forms.ex]: `unquote` has arity 1.

File: lib/elixir/lib/kernel/special_forms.ex (hunk 3 above)
Comment (eksperimental) [inline lib/elixir/lib/kernel/special_forms.ex]: thanks @lexmag for all these good catches.
they have been fixed

File: lib/elixir/lib/macro.ex (hunk 1 above)
Comment (eksperimental) [inline lib/elixir/lib/macro.ex]: fixed

File: lib/ex_unit/lib/ex_unit/doc_test.ex (hunk 2 above)
Comment (eksperimental) [inline lib/ex_unit/lib/ex_unit/doc_test.ex]: fixed

File: lib/elixir/lib/kernel/special_forms.ex (hunk 4 above)
Comment (eksperimental) [inline lib/elixir/lib/kernel/special_forms.ex]: and fixed

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**